sql-plugin/src/main/scala/org/apache/spark/sql/rapids/execution/TrampolineUtil.scala

```diff
@@ -75,4 +76,10 @@ object TrampolineUtil {
   def cleanupAnyExistingSession(): Unit = SparkSession.cleanupAnyExistingSession()
 
   def asNullable(dt: DataType): DataType = dt.asNullable
+
+  /** Return true if the Attribute passed is one of aggregates in the list */
+  def validateAggregate(agg: Attribute): Boolean = {
+    val allAggs = List("min", "max", "avg", "sum", "count", "first", "last")
```

Comment:
Do all of these allow distinct operator on it?

Reply:
Do we need this in Trampoline utils?

Reply:
I couldn't find better place to put this util Method. Creating a new file for this method seemed overkill. Trampoline is only to expose private methods in Spark? I can move this back aggregate.scala file itself if you are okay with it.